common/winobjc.nuproj.common.props

```diff
@@ -18,4 +20,11 @@
     <IsPackable Condition="'$(SkipGetPackageContentsForOtherPlatforms)' == 'true'">false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
It may be worth commenting that this may not work for certain msbuild invocations and refer to the nugetizer issue number right here

Reply:
!! make sure you carry the issue number forward in your squash commit message